test/util/polkadot-api.ts

```diff
@@ -1,48 +1,96 @@
-import { createClient, type PolkadotClient } from "polkadot-api";
+import { createClient, type PolkadotClient, type TypedApi } from "polkadot-api";
 import { WebSocketProvider } from "polkadot-api/ws-provider/node";
 import { relaychain, storagehub } from "@polkadot-api/descriptors";
 
-export type TypesBundle = typeof relaychain | typeof storagehub;
+export type TypesBundle = typeof storagehub | typeof relaychain;
 
-export const getClient = async (endpoint: string, typesBundle: TypesBundle) => {
-  const client = createClient(WebSocketProvider(endpoint));
-  const api = client.getTypedApi(typesBundle);
-  const rt = await api.runtime.latest();
-  return { api, rt, client };
+type StorageHubApi = TypedApi<typeof storagehub>;
+
+export const waitForRandomness = async (api: StorageHubApi, timeoutMs = 60_000) => {
+  process.stdout.write("Waiting for randomness...");
+
+  const waitForValueOrTimeout = (timeoutMs: number): Promise<number> => {
+    return new Promise((resolve, reject) => {
+      const timeout = setTimeout(() => {
+        reject(new Error("Timeout"));
+      }, timeoutMs);
+
+      let valueCount = 0;
+      const subscription = api.query.Randomness.LatestOneEpochAgoRandomness.watchValue(
+        "best"
+      ).subscribe((value) => {
+        valueCount++;
+
+        if (!value) {
+          reject(new Error("Randomness value is undefined"));
+          return;
+        }
+
+        if (valueCount === 2) {
+          clearTimeout(timeout);
+          subscription.unsubscribe();
+          resolve(value[1]);
+        }
+      });
+    });
+  };
+
+  try {
+    const blockHeight = await waitForValueOrTimeout(timeoutMs);
+    if (blockHeight) {
+      process.stdout.write("✅\n");
+      return blockHeight;
+    }
+    process.stdout.write("❌\n");
+    console.error("Timeout reached without receiving a value.");
+  } catch (error) {
+    process.stdout.write("❌\n");
+    console.error("An error occurred:", error);
+  }
 };
 
-export const waitForChain = async (client: PolkadotClient, timeout = 120000) => {
-  console.log(`Waiting a maximum of ${timeout / 1000} seconds for chain to be ready...`);
+export const waitForChain = async (
+  client: PolkadotClient,
+  options?: {
+    timeoutMs?: number;
+    blocks?: number;
+  }
+) => {
+  process.stdout.write(
+    `Waiting a maximum of ${
+      options?.timeoutMs || 60_000 / 1000
+    } seconds for ${await client._request("system_chain", [])} chain to be ready...`
```

Comment:
Any idea why this is giving me the error `Property '_request' does not exist on type 'PolkadotClient'.`

Reply:
`_request` was added in a very recent version of `papi`. It might be you are executing against an old pkg ver